bukkit/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ dependencies {
 
     compileOnly("org.spigotmc:spigot-api:1.19.3-R0.1-SNAPSHOT")
     implementation("io.papermc:paperlib:1.0.8")
-    implementation("dev.rollczi.litecommands:bukkit:2.8.0")
+    implementation("dev.rollczi.litecommands:bukkit:2.8.3")
```

Comment:
Zapoznaj się z zmianami w tych litecommandach, sprawdź czy wszystko działa na serwerze.

W skrócie mogę powiedzieć że nie jest już wymagana adnotacja @name, teraz litecommandy pobierają nazwę z nazwy argumentu 

Np. 

```java 
void execute(Player player, @Arg @Name("messages") String text) 
``` 
Powodowało że nazwą argumentu będzie "messages"

Teraz nie musisz tego robić i wystarczy po prostu String nazwać messgaes zamiast text. Oczywiście to jest tylko przykład i sam musisz sprawdzić czy taka zmiana będzie tutaj potrzebna

Reply:
shadowJar normalnie się skompilował, zaraz przetestuje

Reply:
Tested, działa.